specs/phase0/beacon-chain.md

```diff
@@ -1229,7 +1230,12 @@ def is_valid_genesis_state(state: BeaconState) -> bool:
 
 ### Genesis block
 
-Let `genesis_block = BeaconBlock(state_root=hash_tree_root(genesis_state))`.
```

Comment:
Probably we should keep this line and make the following modification:
- Let `genesis_block = get_genesis_block_from_genesis_state(genesis_state)`, where:
```python
 def get_genesis_block_from_genesis_state(genesis_state: BeaconState) -> BeaconBlock:
     return BeaconBlock(state_root=hash_tree_root(genesis_state))
 ```

Reply:
agreed

Reply:
also agreed